src/main/resources/javascript/getAllElementsByPath.js

```diff
@@ -52,6 +52,19 @@ function addCoordinates(extractedAttributes, node) {
 	}
 }
 
+function getDisabled(node) {
```

Comment:
Like `isShown` or `isNonEmptyTextNode`:

```suggestion
function isDisabled(node) {
```